_includes/version-warning

```diff
@@ -0,0 +1,24 @@
+<style type="text/css">
+    .disclaimer {
+        border: 1px solid #000000;
+        border-radius: 10px;
+        margin: 10px 50px 40px;
+        padding: 20px;
+        background-color: #ffbc41;
+    }
+    .heading {
+        font-weight: 700;
+        font-size: 1.4em;
+        padding: 0 0 .3em;
+    }
+</style>
+
+<div class="disclaimer">
+    <div class="heading">Attention</div>
+    <p>
+        This Chapter of the MEI Guidelines is outdated and reflects the older state of the MEI v3 release. 
+        It may contradict the current state of the MEI specifications as documented in the Elements, 
+        Attribute Classes, Model Classes, Data Types and Macro Groups sections. The Community is currently 
+        working to update these Guidelines. Of course, help is greatly appreciated. In that case, please 
```

Comment:
Of course 
```suggestion
        working to update these Guidelines. Of course, help is greatly appreciated. In case you would like to contribute, please 
```